src/modules/auth/dto/login.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsEmail, IsNotEmpty, IsString } from 'class-validator';
+
+export class LoginDto {
+  @IsNotEmpty()
+  @IsEmail({}, { message: 'Please enter correct email address' })
```

Comment:
Not sure if the message is required as the validation already does this

Reply:
Removed